packages/moonstone/ExpandablePicker/ExpandablePicker.js

```diff
@@ -71,9 +71,9 @@ const ExpandablePickerBase = kind({
 		decrementAriaLabel: PropTypes.string,
 
 		/**
-		 * A custom icon for the decrementer. All strings supported by [Icon]{Icon} are
+		 * A custom icon for the decrementer. All strings supported by [Icon]{@link moonstone/Icon} are
```

Comment:
Can you make this `moonstone/Icon.Icon`?

Reply:
:sadpanda: